### ISSUE TITLE
The recent episodes component is nested in the main landmark element

### ISSUE THREAD

**Issue by jroebu14** [bbc/simorgh#8591]
**Describe the bug**
Acceptance critera states:

> this component will not be nested within another landmark e.g. the main landmark”

This is not the case for audio and video recent episodes - they are positioned in the main element.

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://www.test.bbc.com/afrique/bbc_afrique_radio/w172xr8nm92cx20 and https://www.test.bbc.com/afrique/bbc_afrique_tv/tv_programmes/w13xttmz
2. View source and observe the recent episodes component is positioned inside the main element

**Expected behaviour**
The component should be positioned outside of the main landmark.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**
